.gitignore

```diff
@@ -37,13 +68,9 @@ coverage.xml
 node_modules/
 bower_components/
 .grunt/
+vendor/
+static/vendor/
 src/vendor/
-src/static/fonts/
```

Comment:
I have them locally there. Not sure why they got removed from that location/ignored. I'll look back at the `.gitignore` blame in a sec. But if you look at `grunt:copy`, that was the original intent.